src/test/java/com/github/bot/curiosone/core/workflow/LogicTest.java

```diff
@@ -16,6 +16,15 @@ public void testAnswer() throws IOException {
     assertThat(msg.getMessage()).isEqualTo("Sorry my head hurts, what were we talking about?");
     assertThat(msg.getScope()).isEmpty();
 
+    msg = Logic.talk(null);
+    assertThat(msg.getMessage()).isEqualTo("Sorry my head hurts, what were we talking about?");
+    assertThat(msg.getScope()).isEmpty();
+
+    msg = Logic.talk(new Message("Sorry my head hurts, what were we talking about? "
+        + "Sorry my head hurts, what were we talking about?", "", ""));
+    assertThat(msg.getMessage())
+        .containsIgnoringCase("Sorry my head hurts, what were we talking about?");
```

Comment:
I would remove this particular test. Is not always true that the answer should be that. We could make it more smarter in the future.